### DISCUSSION TITLE
Bug in GMSH interface when the element is a 20 node brick ?

### DISCUSSION THREAD

**Discussion by andprad74** [OpenRadioss/OpenRadioss#2809] [Users]
Hello,
and thanks in advance.
I use the following workaround in order to create the groups for openradioss:

Salome ----> GMSH ---->Radioss Mesh Interface by PaulAltair ---->name_file_0000.rad.
When the conversion involve a brick 20 nodes, the Starter evidence an error.
The same mesh, in Ls-Prespost, evidence a mistake in the conversion.
Probably is a bug.
Thanks by.
Andrea

![movie_000_00](https://github.com/user-attachments/assets/538fd342-00b1-4bc9-8bf1-f13b11ba1ade)[bug_gmsh.zip](https://github.com/user-attachments/files/16225271/bug_gmsh.zip)

**Comment by PaulAltair** [comment]
Thanks for catching, I have submitted a modification to the 2nd order node ordering in Gmsh radioss profile, if you use the 'development build' of Gmsh this should be resolved, if you can try it for your use case and let me know?

**Comment by andprad74** [comment]
Hi Paul,
finally i have had time to check and now it works!
For my side the topic can be closed as solved.
Thanks a lot.
Andrea.
![ok_it_works](https://github.com/user-attachments/assets/bdd21413-e08f-4843-94f2-c477a04ec989)

**Reply by PaulAltair** [reply]
Great, thanks for the feedback

**Reply by MarianBulla** [reply]
Hi @andprad74,

this is great, thank you @PaulAltair for solving this, and thank you very much @andprad74 for confirmation and for your interest in OpenRadioss.

Thanks and best regards,
Marian
